dotnet/src/SemanticKernel.Abstractions/Models/PromptFunctionModel.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.AI;
+
+namespace Microsoft.SemanticKernel.Models;
+
+/// <summary>
+/// Prompt model files.
+/// </summary>
+public sealed class PromptFunctionModel
+{
+    /// <summary>
+    /// Name of the kernel function.
+    /// </summary>
+    [JsonPropertyName("name")]
+    public string Name { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Format of the prompt template e.g. f-string, semantic-kernel, handlebars, ...
+    /// </summary>
+    [JsonPropertyName("template_format")]
+    public string TemplateFormat { get; set; } = Microsoft.SemanticKernel.TemplateEngine.PromptTemplateConfig.SemanticKernelTemplateFormat;
```

Comment:
nit. Avoid long lines (fully qualified names if no conflict exists)

``` 
using Microsoft.SemanticKernel.TemplateEngine;
```

```suggestion
    public string TemplateFormat { get; set; } = PromptTemplateConfig.SemanticKernelTemplateFormat;
```